packages/fxa-payments-server/src/components/PaymentMethodHeader/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Localized } from '@fluent/react';
+import { Plan } from 'fxa-shared/subscriptions/types';
+import React from 'react';
+import useValidatorState from '../../lib/validator';
+import { Form } from '../fields';
+import { PaymentConsentCheckbox } from '../PaymentConsentCheckbox';
+
+export type PaymentMethodHeaderProps = {
+  plan: Plan;
+  onClick: (event: React.MouseEvent<HTMLInputElement>) => void;
+  prefix?: string;
+};
+
+export const PaymentMethodHeader = ({
+  plan,
+  onClick,
+  prefix = '',
+}: PaymentMethodHeaderProps) => {
+  const checkboxValidator = useValidatorState();
+  const headerMarginTop = '40px';
```

Comment:
Why the inline style?  I don't think that's something we've been doing in this frontend.

Reply:
I created the `index.scss` file and added the class in there.